sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedSiteCreation

```diff
@@ -0,0 +1,78 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedsitecreation
+applicable: SharePoint Online
+title: Get-SPORestrictedSiteCreation
+schema: 2.0.0
+author: vgaddam-pm
+ms.author: vgaddam
+ms.reviewer:
+---
+
+# Get-SPORestrictedSiteCreation
+
+## SYNOPSIS
+
+This cmdlet allows SharePoint administrators to check the current configuration of the restricted site creation feature.
+
+## SYNTAX
+
+```powershell
+Get-SPORestrictedSiteCreation
+   [-SiteType <RestrictedSiteCreationSiteType>]
+```
+
+## DESCRIPTION
+
+This command obtains the current configuration information for the restricted site creation feature, including whether it is enabled, the current mode, and the current policies.
+
+> [!Important]
+>You must use version 16.0.25513 (published November 2024) or later of the [SharePoint Online Management Shell](https://www.microsoft.com/en-us/download/details.aspx?id=3558) for these commands to function properly. Earlier versions do not have the current list of site types and will not operate correctly.
```

Comment:
remove this part -- it's not required, and we prefer not to keep such info since it becomes stale pretty soon.

admins already understand to install latest module if a cmdlet is missing from their install shell version

Reply:
During private preview, we realized we needed to redefine the values of the RestrictedSiteCreationSiteType enum before GA. This change will ship with the November 2024 version of TAPS, but unfortunately means that older versions of TAPS use the wrong enum and will cause confusion for customers.

Reply:
The problem is you'll now have to come back and update this documentation at some time to remove this note.

And oh, we don't yet know what build number for November is -- until the PM picks it we shouldn't assume it'll be 25513